IPython/html/static/edit/less/edit.less

```diff
@@ -1,3 +1,10 @@
+.header_edit {
+}
+
+.site_edit {
+    height: 88.5%;
```

Comment:
site height shouldn't be a fixed percentage. That gets the wrong value as the height of the window changes. It should probably be `calc(100vh - header_height)` or something to that effect.